### PR TITLE
Make a Outgoing Call to connect Numbers

### PR DESCRIPTION
**Pull request by sameera222**
This Pull Request Will: 

- [x] Call API for connecting two numbers
- [x] Add contact list to the App

### CODE REVIEW

File: apps/Dataman/babel.config.js [hunk 1]
```diff
@@ -4,6 +4,21 @@ module.exports = function (api) {
     presets: ["babel-preset-expo"],
     plugins: [
       "nativewind/babel",
+      [
+        "module:react-native-dotenv",
+        {
+          envName: "APP_ENV",
```
Comment (sutyum) [inline apps/Dataman/babel.config.js:12]: ?

File: apps/Dataman/src/Utils/colors.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
+export const getColorsByLetter = (letter: string) => {
```
Comment (sutyum) [inline apps/Dataman/src/Utils/colors.ts]: why we need this file?

**Review by sutyum — APPROVED** [review]

File: apps/Dataman/babel.config.js (hunk 1 above)
Comment (sameera222) [reply apps/Dataman/babel.config.js:12]: It allows to use environment variables in React Native application.

File: apps/Dataman/src/Utils/colors.ts (hunk 2 above)
Comment (sameera222) [reply apps/Dataman/src/Utils/colors.ts]: To show color behind profile.

File: apps/Dataman/babel.config.js (hunk 1 above)
Comment (sutyum) [reply apps/Dataman/babel.config.js:12]: put a reference link to the blog/docs you followed. Put it as a comment in the code at this position as well as here.

File: apps/Dataman/src/Utils/colors.ts (hunk 2 above)
Comment (sutyum) [reply apps/Dataman/src/Utils/colors.ts]: Why is this color behind profile needed or useful?

File: apps/Dataman/babel.config.js (hunk 1 above)
Comment (sameera222) [reply apps/Dataman/babel.config.js:12]: https://docs.expo.dev/guides/environment-variables/     
https://blog.logrocket.com/understanding-react-native-env-variables/ 
These are the link of expo documentation and blog from where I have taken a reference.

File: apps/Dataman/src/Utils/colors.ts (hunk 2 above)
Comment (sameera222) [reply apps/Dataman/src/Utils/colors.ts]: This file has been removed now

**Pull request merged**